### DISCUSSION TITLE
Welcome to describo-online Discussions!

### DISCUSSION THREAD

**Discussion by marcolarosa** [Arkisto-Platform/describo-online#47] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by juriroemer** [comment]
Hi all,

I'm Juri, currently working at the University of Münster (Germany) on a project called Sciebo RDS, where we connect EFSS Systems to research data service repositories (such as Zenodo, OSF, …). We embed Describo to enrich data with metadata before it is uploaded to the repositories, making sure it has (at least) all the metadata required by the repositories.

Here I'm mostly trying to bring small UI/bug fixes and improvements into the code and also maintaining our own fork of Describo, which contains a few visual adjustments for our own purposes (technically it is 100% the same, though).

Happy to be here and excited to have a discourse with all of you!

🚀
